test/integrations/destinations/clevertap/dataDelivery/business.ts

```diff
@@ -0,0 +1,219 @@
+              'Content-Type': 'application/json',
+            },
+            JSON: {
+              d: [
+                {
+                  identity: 'anon-id-new',
+                  type: 'event',
+                  evtName: 'Web Page Viewed: Rudder',
+                  evtData: {
+                    title: 'Home',
+                    path: '/',
+                  },
+                },
+              ],
+            },
+            endpoint: 'https://api.clevertap.com/1/upload/test2',
+          },
+          [generateMetadata(123)],
+        ),
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            status: 401,
+            message: 'Request failed  with status: 401',
+            response: [
+              {
+                metadata: generateMetadata(123),
+                error: '{"status":"fail","error":"Invalid Credentials","code":401}',
+                statusCode: 401,
+              },
+            ],
+            statTags: {
+              destType: 'CLEVERTAP',
+              destinationId: 'default-destinationId',
+              errorCategory: 'network',
+              errorType: 'aborted',
+              feature: 'dataDelivery',
+              implementation: 'native',
+              module: 'destination',
+              workspaceId: 'default-workspaceId',
+            },
+          },
+        },
+      },
+    },
+  },
+  {
+    id: 'clevertap_business_2',
+    scenario: 'business',
+    successCriteria: 'should return 401 status code with error message',
+    name: 'clevertap',
+    description: '[business]:: event failed due to invalid credentials',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v1',
+    input: {
+      request: {
+        body: generateProxyV1Payload(
+          {
+            params,
+            headers: {
+              'X-CleverTap-Account-Id': '476550467',
+              'X-CleverTap-Passcode':
+                'fbee74a147828e2932c701d19dc1f2dcfa4ac0048be3aa3a88d427090a59dc1c0fa002f1',
+              'Content-Type': 'application/json',
+            },
+            JSON: {
+              d: [
+                {
+                  identity: 'anon-id-new',
+                  type: 'event',
+                  evtData: {
+                    title: 'Home',
+                    path: '/',
+                  },
+                },
+              ],
+            },
+            endpoint: 'https://api.clevertap.com/1/upload/test3',
+          },
+          [generateMetadata(123)],
+        ),
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            status: 400,
+            message: 'Request failed  with status: 200',
+            response: [
+              {
+                metadata: generateMetadata(123),
+                error: '{"status":"fail","processed":0,"unprocessed":[]}',
```

Comment:
Is it possible to add a partial failure response case too?
like 
`{"status":"fail","processed":1,"unprocessed": 3, reason[ { some error for some record}]}`